optimizely/helpers/enums.py

```diff
@@ -90,14 +90,20 @@ class NotificationTypes(object):
       TRACK notification listener has the following parameters:
       str event_key, str user_id, dict attributes (can be None), event_tags (can be None), Event event
       DECISION notification listener has the following parameters:
-      DecisionInfoTypes type, str user_id, dict attributes (can be None), dict decision_info
+      DecisionNotificationTypes type, str user_id, dict attributes, dict decision_info
   """
   ACTIVATE = "ACTIVATE:experiment, user_id, attributes, variation, event"
   DECISION = "DECISION:type, user_id, attributes, decision_info"
   TRACK = "TRACK:event_key, user_id, attributes, event_tags, event"
 
 
-class DecisionInfoTypes(object):
-  EXPERIMENT = "experiment"
+class DecisionNotificationTypes(object):
```

Comment:
nit. Lets alphabetize class names here.

Also, conventionally we do single quotes for all strings.